lib/models/User.dart

```diff
@@ -0,0 +1,21 @@
+class User{
```

Comment:
The file name should be in  `lowercase`. Have a look at  https://dart.dev/guides/language/effective-dart/style

Reply:
Also the package name should be named in `lowercase_with_underscores` format.